runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11790,53 +11788,130 @@ static bool inlineIsAssignableFrom(TR::Node *node, TR::CodeGenerator *cg)
    return true;
    }
 
+static TR::SymbolReference *getClassSymRefAndDepth(TR::Node *classNode, TR::Compilation *comp, int32_t &classDepth)
+   {
+   classDepth = -1;
+    // recognizedCallTransformer adds another layer of aloadi
+   // need to loop find non-aloadi child
+   while (classNode->getOpCodeValue() == TR::aloadi && classNode->getNumChildren() > 0 && classNode->getFirstChild()->getOpCodeValue() == TR::aloadi)
+      {
+      classNode = classNode->getFirstChild();
+      }
+
+   if ((classNode->getOpCodeValue() != TR::aloadi) || (classNode->getSymbolReference() != comp->getSymRefTab()->findJavaLangClassFromClassSymbolRef()) ||
```

Comment:
Please reformat this to have return statement in new line - to make this more readable.